bigquery/deploy/README.md

```diff
@@ -46,19 +46,31 @@
     * In the project with the BigQuery dataset, make the service account a
     BigQuery Data Viewer.
   * Create cluster
-    * Go to https://console.cloud.google.com/kubernetes/list and click `Create Cluster`
-    * Change name to `elasticsearch-cluster`
-    * Set zone
-      * If you have [deployed the UI server and/or API server](https://github.com/DataBiosphere/data-explorer/tree/master/deploy),
-        select a zone in the same region as UI/API server. (Search for `Region`
-        in the App Engine Dashbaord.)
-      * If you not not yet deployed the UI/API servers, make a note what region
-        your cluster is in. Later when you run `gcloud app create`, select this
-        region. This is important because App Engine app regions cannot be
-        changed after `gcloud app create`. (The Elasticsearch deployement uses
-        [Internal Load Balancing](https://cloud.google.com/kubernetes-engine/docs/how-to/internal-load-balancing),
-        so the API server will only be able to talk to Elasticsearch if it's in
-        the same region.)
+    * Run:
+    ```
+    gcloud container clusters create elasticsearch-cluster --num-nodes=1 --service-account=indexer@<project>.iam.gserviceaccount.com --zone=ZONE
+    gcloud container node-pools create master-pool --cluster=elasticsearch-cluster --num-nodes=1 --node-labels=type=master --machine-type=n1-standard-1 --zone=ZONE
+    gcloud container node-pools create data-pool --cluster=elasticsearch-cluster --num-nodes=1 --node-labels=type=data --machine-type=n1-standard-2 --disk-type=pd-ssd --zone=ZONE
+    gcloud container node-pools create client-pool --cluster=elasticsearch-cluster --num-nodes=1 --node-labels=type=client --machine-type=n1-standard-4 --zone=ZONE
+    ```
+    * Horizontal Scaling: We have a 1:1 mapping of Kubernetes node : GCE VM to 
```

Comment:
I guess this should wait till your next PR -- If there is large dataset like NHS, can you give guidance on how to scale, eg increase CPU and/or memory, and in which pool?

Reply:
Added guidance on how to scale and what aspects are important. I will also go back and try to optimize NHS again given more recent changes to this PR. I expect it will mostly require vertical scaling of cpu/memory for data nodes, and perhaps a bit for the coordinating nodes as well. 

As a note, I changed the use of `client node` to `coordinating node` throughout, as this naming change was done in ES 5 (which we are way beyond now): https://discuss.elastic.co/t/client-node-and-coordinating-nodes/82384/2